examples/providerconfig/secret.yaml.tmpl

```diff
@@ -7,6 +7,9 @@ type: Opaque
 stringData:
   credentials: |
     {
-      "username": "admin",
-      "password": "t0ps3cr3t11"
+      "cloud_api_key": "admin",
+      "cloud_api_secret": "t0ps3cr3t11",
+      "kafka_api_key": "kafka_admin",
+      "kafka_api_secret": "P@55w0rd",
+      "kafka_rest_endpoint": "https://abc-12345z.region.provider.confluent.cloud:443"
```

Comment:
I noticed the confluent terraform docs also have a `kafka_id` field in the provider auth block for single-cluster management. Did you happen to know anything about that and if it's required or not?

ref: https://registry.terraform.io/providers/confluentinc/confluent/latest/docs#static-credentials

Reply:
That's a good question.

I noticed `kafka_id` on the terraform docs as well, but the specific error I was seeing with `KafkaACL`s (and what @drneo-mehdi was seeing with `KafkaTopic`s) only indicated that the key and secret were missing:

`cannot run refresh: refresh failed: error reading Kafka Topic: one of (provider.kafka_api_key, provider.kafka_api_secret), (KAFKA_API_KEY, KAFKA_API_SECRET environment variables) or (resource.credentials.key, resource.credentials.secret) must be set:`

That being said, I could try adding a `kafka_id`/`kafkaID` field to the ProviderConfig secret just in case I'm missing something.  Let me know if you think that's warranted here.

Reply:
I personally haven't ran into any scenarios where I needed the kafka ID, but my team is only using the multi-cluster authentication method, which is why we never prioritized the other authentication method using kafka_rest_endpoint.

I'd say as long as it works we can get this merged and look into what role kafka_id plays at a later point 👍